spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapPropertySource.java

```diff
@@ -64,37 +61,26 @@ protected static Environment createEnvironmentWithActiveProfiles(String[] active
 	}
 
 	protected static String getName(String name, String namespace) {
-		return new StringBuilder().append(PREFIX).append(Constants.PROPERTY_SOURCE_NAME_SEPARATOR).append(name)
-				.append(Constants.PROPERTY_SOURCE_NAME_SEPARATOR).append(namespace).toString();
+		return PREFIX + PROPERTY_SOURCE_NAME_SEPARATOR + name + PROPERTY_SOURCE_NAME_SEPARATOR + namespace;
```

Comment:
a slight change to drop `StringBuilder`, the compiled code would look exactly the same anyway.